.bsb.lock

```diff
@@ -0,0 +1 @@
+19483
```

Comment:
Should we include `.bsb.lock`

Reply:
Oh sorry probably not. Think it was introduced as part of the move to reason 3

Reply:
I've removed it